### ISSUE TITLE
Mail Attachment Error

### ISSUE THREAD

**Issue by mazvi** [rathena/rathena#5927]
* **rAthena Hash**: 16b4cbf

* **Client Date**: 20200401

* **Server Mode**: RENEWAL

* **Description of Issue**: Mail Attachment Error: Please empty your inventory.
  * Result:  Cannot download attachment one by one without relogin
  * Expected Result:  Can download attachment one by one without relogin
  * How to Reproduce:  Send 2 or more email to your character with attachment, then download one by one. The storage is very free.
  * Official Information: 
  

![001](https://user-images.githubusercontent.com/8316103/117583589-23151f00-b132-11eb-9491-04fc89ba8202.jpg)

* **Modifications that may affect results**: 
```
// Delay to allow user resend new mail (default & minimum is 1000)
mail_delay: 300000

// Amount of mails a user can send a day.
// Default: 100
// 0 = Unlimited
mail_daily_count: 100
```
  
  

Maybe related: https://github.com/rathena/rathena/commit/f77cb8a37ba4441742b6097d7b778df05851a619

**Comment by Lemongrass3110** [comment]
And you are sure that you applied https://github.com/rathena/rathena/commit/55ca79961da77d1ba6c244393de0101709e7bc0d correctly?

**Comment by mazvi** [comment]
Yes.. i only use git pull every time not using one by one patch..

**Comment by admkakaroto** [comment]
Can i confitm this.

**Comment by RadianFord** [comment]
I run a test of this on my server, i cant reproduced it sending multiple mail with item attachments. 
![Untitled](https://user-images.githubusercontent.com/9658425/117589425-3f876c00-b0de-11eb-9230-5fc1fe8de70f.png)

**Comment by pigati23** [comment]
That also happens to me sometimes.

**Comment by mazvi** [comment]
> 
> 
> I run a test of this on my server, i cant reproduced it sending multiple mail with item attachments.
> ![Untitled](https://user-images.githubusercontent.com/9658425/117589425-3f876c00-b0de-11eb-9230-5fc1fe8de70f.png)

are u using linux GCC Compiler?

**Comment by RadianFord** [comment]
> > I run a test of this on my server, i cant reproduced it sending multiple mail with item attachments.
> > ![Untitled](https://user-images.githubusercontent.com/9658425/117589425-3f876c00-b0de-11eb-9230-5fc1fe8de70f.png)
> 
> are u using linux GCC Compiler?

yes, on Ubuntu

**Comment by mazvi** [comment]
> 
> 
> That also happens to me sometimes.

I think this happened when we received attachments from the attendance once, then received attachments from others. Then error occurred. can you @RadianFord test by receiving an attachment from the attendance once, then downloading another attachment?

**Comment by RadianFord** [comment]
> > That also happens to me sometimes.
> 
> I think this happened when we received attachments from the attendance once, then received attachments from others. Then error occurred. can you @RadianFord test by receiving an attachment from the attendance once, then downloading another attachment?

I disabled my attendance system, but i can try and see if its producing this issue.

**Comment by mazvi** [comment]
What @pigati23 said is true, it happens sometimes. Even today I tried normal, but yesterday it was an error.

**Comment by Lemongrass3110** [comment]
This was fixed in d25ec40 as well
